pkg/platform/provider/baremetal/cluster/kubeadm.go

```diff
@@ -215,9 +216,6 @@ func (p *Provider) getControllerManagerExtraArgs(c *v1.Cluster) map[string]strin
 		"cluster-cidr":             c.Spec.ClusterCIDR,
 		"service-cluster-ip-range": c.Status.ServiceCIDR,
 	}
-	for k, v := range c.Spec.ControllerManagerExtraArgs {
-		args[k] = v
-	}
```

Comment:
why remove above lines?

Reply:
it's redundant. the assignment has been done in the next line.